src/lib/djerba/extract/maf_annotater.py

```diff
@@ -0,0 +1,49 @@
+"""Functions of annotating MAF variants with oncoKB"""
+
+import os
+import logging
+from djerba.util.subprocess_runner import subprocess_runner
+from djerba.util.logger import logger
+
+class maf_annotater():
```

Comment:
I've dealt with the logger part, but about the CNA/fusions: when we'd talked in the office, you'd recommended treating MAF separately from CNA/fusion because (if I remember correctly) the latter are writing to temp, while MAF is saving to report dir/ , does that still feel like a concern or just move all the oncokb functions to the new class?

Reply:
Having looked more closely at the code, I think all in the same class is better. As I say, I'll draft an example.